internal/provider/data_source_hvn_route.go

```diff
@@ -52,6 +52,7 @@ func dataSourceHVNRoute() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			// TODO add updated_at after public API returns updated_at for HVN route
```

Comment:
This work has been recently merged (https://github.com/hashicorp/cloud-network/pull/526), we will need to release a new version of the SDK in order to leverage the change, my suggestion is to do it in a follow up PR